use-cases/vehicle-routing/src/main/resources/META-INF/resources/app.js

```diff
@@ -336,6 +337,116 @@ function renderTimelines(routePlan) {
     }
 }
 
+function analyze() {
```

Comment:
Please consider breaking this long function into shorter ones to increase the readability.

Reply:
Refactored.